### DISCUSSION TITLE
Photon Import from Nominatim is Interrupted

### DISCUSSION THREAD

**Discussion by Imane-ux** [komoot/photon#853] [Q&A]
There was a connection loss to my nominatim database when photon was in the process of importing data from it. Is there a way or command line to allow the process to resume its progress from where it was left off, instead of doing a new import from scratch.

For example, nominatim has (nominatim import --continue indexing) command line in case the import process was accidentally killed or connection is lost. Is there a similar solution for photon or a workaround solution please.

Thanks,

**Comment by lonvia** [comment]
No, the import cannot be continued. Photon does not keep track of what has been done already.
